tip-wlan/charts/wlan-cloud-graphql-gw/templates/ingress.yaml

```diff
@@ -32,6 +32,12 @@ spec:
     - host: {{ .host | quote }}
       http:
         paths:
+        {{- if $.Values.ingress.alb_https_redirect }}
+          - path: /*
+            backend:
+              serviceName: ssl-redirect
+              servicePort: use-annotation
```

Comment:
In general, we define properties in values.yaml, not in the resource's yaml

Reply:
I agree, although here if you enable alb redirection, you should not put anything different there, parametrizing here seems excessive to me.
The whole loop over Values.ingress.hosts needs to be rewritten though, paths need to be passed from values but thats not possible now due to $fullName and $svcPort being hard coded inside the loop, subject to a dedicated refactoring.
Please see also discussion here https://github.com/Telecominfraproject/wlan-cloud-helm/pull/21#discussion_r519735566